src/main/java/com/uid2/admin/vertx/service/EncryptionKeyService.java

```diff
@@ -157,11 +169,11 @@ private void handleKeyList(RoutingContext rc) {
 
     private void handleRotateMasterKey(RoutingContext rc) {
         try {
-            final RotationResult result = rotateKeys(rc, masterKeyActivatesIn, masterKeyExpiresAfter,
-                    s -> s == Const.Data.MasterKeySiteId || s == Const.Data.RefreshKeySiteId);
+            final RotationResult masterKeyResult = rotateKeys(rc, masterKeyActivatesIn, masterKeyExpiresAfter,
+                s -> s == Const.Data.MasterKeySiteId || s == Const.Data.RefreshKeySiteId);
```

Comment:
then master key and refresh key will have the same expiry?

Reply:
Yes, that's the original logic and idk if we plan to change that.

Reply:
Just checking if it's intentional. Good.